manifests/config/global/framework.pp

```diff
@@ -4,20 +4,46 @@
 
 # == Class rundeck::config::global::framework
 #
-# This private class is called from rundeck::config used to manage the framework properties of rundeck
+# This private class is called from rundeck::config and
+# used to manage the framework properties of rundeck
 #
 class rundeck::config::global::framework(
-  $properties_dir   = $rundeck::config::properties_dir,
-  $user             = $rundeck::config::user,
-  $group            = $rundeck::config::group
-
+  $properties_dir = $rundeck::config::properties_dir,
```

Comment:
@jyaworski ,can we not change code as this way? codes in fact are same, but it is annoying to spend time on it, because it looks to be changed. There are a lot in this pull request, please fix all of them.